Lib/ast.py

```diff
@@ -643,6 +643,14 @@ def main():
                              'column offsets')
     parser.add_argument('-i', '--indent', type=int, default=3,
                         help='indentation of nodes (number of spaces)')
+    parser.add_argument('--feature-version',
+                        type=str, default=None, metavar='VERSION',
+                        help='minor version (int) or 3.x tuple (e.g., 3.10)')
```

Comment:
let's always require a full version. this can potentially be confusing in the future 4 version.